src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -73,16 +73,18 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
         } catch (error) {
             const parsedError: IParsedError = parseError(error);
 
-            if (parsedError.errorType !== 'UserCancelledError') {
+            if (parsedError.message.match(/password authentication failed for user/) || parsedError.errorType === 'UserCancelledError') {
```

Comment:
Can you check against errorType instead? This won't work for other languages, and is prone to problems if they even slightly change the message